lib/src/widgets/profile_header.dart

```diff
@@ -176,8 +178,11 @@ class ProfileAvatar extends StatelessWidget {
           maxHeight: editUserAvatarImageMaxSize,
         );
         if (croppedAvatar != null) {
+          _isImageAvailable = true;
           ProfileData.of(context).imageActionCallback(croppedAvatar.path);
         }
+      } else {
+        _isImageAvailable = false;
```

Comment:
In my test the extra adjustment of `_isImageAvailable` wasn't required. Please try if everything works without this. Otherwise LGTM.

Reply:
Removed and it works :)